composer.json

```diff
@@ -29,12 +29,17 @@
         {
             "type": "vcs",
             "url": "https://github.com/JellyBellyDev/phpspec-data-provider-extension"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/ricardoboss/easysvg"
         }
     ],
     "require": {
         "php": ">=7.4",
         "ext-gd": "*",
         "ext-simplexml": "*",
+        "kartsims/easysvg": "^2.0",
```

Comment:
Do we really need extra dependency? How svg rendering is implemented in other badge styles without it?

Reply:
In https://github.com/ekfuhrmann/badge-generator it is implemented using opentype.js. We need some way to render a font in SVG, since not everyone has the required fonts installed on their systems. If you another way, let me know!

Reply:
No problem for me to new deps but i prefer to use the standard font as talk by @antonkomarev here: https://github.com/badges/poser/pull/86#issuecomment-1023287355